var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -615,7 +621,10 @@ def enable_or_disable(variant, keyword="USE", var=None):
             env.set("WITH_BLAS", "generic")
 
         # Don't use vendored third-party libraries when possible
-        env.set("BUILD_CUSTOM_PROTOBUF", "OFF")
+        if self.spec.satisfies("~internal-protobuf"):
```

Comment:
This double negative is a bit confusing, can we make it "if +, ON, else, OFF"?

Reply:
Sure